cli/command/image/build.go

```diff
@@ -227,13 +239,56 @@ func runBuild(dockerCli *command.DockerCli, options buildOptions) error {
 		buildCtx, err = archive.TarWithOptions(contextDir, &archive.TarOptions{
 			Compression:     compression,
 			ExcludePatterns: excludes,
-			IncludeFiles:    includes,
```

Comment:
This is the only issue I wasn't sure about. What do you think the comment should say?